consent/strategy_default.go

```diff
@@ -120,18 +120,18 @@ func (s *DefaultStrategy) authenticationSession(ctx context.Context, _ http.Resp
 	return session, nil
 }
 
-func (s *DefaultStrategy) requestAuthentication(ctx context.Context, w http.ResponseWriter, r *http.Request, ar fosite.AuthorizeRequester) (err error) {
+func (s *DefaultStrategy) requestAuthentication(ctx context.Context, w http.ResponseWriter, r *http.Request, ar fosite.AuthorizeRequester, requestURL *url.URL, deviceFlowID string) (err error) {
```

Comment:
Perhaps we should find another way to pass these new params to the `forwardAuthenticationRequest` method.